app/src/main/java/ch/epfl/sdp/blindly/recording/RecordingActivity.kt

```diff
@@ -61,11 +67,11 @@ class RecordingActivity : AppCompatActivity() {
     override fun onStop() {
         super.onStop()
         mediaRecorder.release()
-        mediaPlayer?.release()
-        mediaPlayer = null
+        deleteTempRecordings()
+        adapter.recordList = ArrayList()
```

Comment:
Is it really necessary to empty the adapter's recordList ? The adapter is created from scratch everytime you launch the activity, isn't it? Or am I missing something ?

Reply:
I think it is better to do it (think of it as a memory deallocation). But yes, it is created from scratch everytime you launch it.